frontend/src/metabase/admin/datamodel/selectors.js

```diff
@@ -3,17 +3,17 @@ import { createSelector } from 'reselect';
 import { computeMetadataStrength } from "metabase/lib/schema_metadata";
 
 
-const segmentsSelector         = state => state.datamodel.segments;
-const metricsSelector          = state => state.datamodel.metrics;
+const segmentsSelector         = (state, props) => state.datamodel.segments;
```

Comment:
I see that makes sense then. 

Although I think this might be a good use case for rest params and then destructuring into the selector args if we ever want to refactor this. I've never actually used it before though so not sure if there might be complications.